lib/card.rb

```diff
@@ -0,0 +1,18 @@
+class Card
+
+  attr_reader :id
+
+  def initialize(id, attributes)
+    @id = id
+    @attributes = attributes
+  end
+
+  def method_missing(method_name, *args)
```

Comment:
~~~ruby
[9] pry(main)> card
=> #<Card:0x007feb6596e7d8 @attributes={:symbol=>:oval, :number=>:one, :color=>:purple, :shading=>:open}, @id=62>
[10] pry(main)> card.respond_to? :symbol
=> false
[11] pry(main)> card.symbol
=> :oval
~~~

Yeah I think so since this makes no sense. While `method_missing` is neat, I think the cost here is a little to high and this smells a lot like needless metaprogramming.